collections/guides/quickstarts/quickstart_java.md

```diff
@@ -0,0 +1,234 @@
+
+```java
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse.BodyHandlers;
+
+public class BTQuickstart {
+
+    public static void main(String[] args) {
+        try {
+            // Initialize the HttpClient
+            var client = HttpClient.newHttpClient();
+
+            // JSON message formatted as a string
+            var message = "{\"type\": \"token\", \"data\": \"foo\"}";
+
+            var createRequest = HttpRequest.newBuilder()
+                .uri(URI.create("https://api.basistheory.com/tokens"))
+                .POST(BodyPublishers.ofString(message))
+                .header("Content-type", "application/json")
+                .header("Accept", "application/json")
+                // Enter your API key here!
+                .header("BT-API-KEY", "key_E6gVJEn1DRT5m6YHs55Dsa")
+                .build();
+
+            // Create a new token
+            var createResponse = client.send(createRequest, BodyHandlers.ofString());
+            var token = createResponse.body();
+
+            // Token response
+            System.out.println("Create a token:");
+            System.out.println(token);
+
+            // Parse token id from response
+            var tokenArray = token.split("\"");
+            var tokenId = tokenArray[3];
+
+            // Get token
+            var getRequest = HttpRequest.newBuilder()
+                .uri(URI.create(String.format("https://api.basistheory.com/tokens/%s", tokenId)))
+                .GET()
+                .header("Content-type", "application/json")
+                .header("Accept", "application/json")
+                .header("BT-API-KEY", "key_E6gVJEn1DRT5m6YHs55Dsa")
+                .build();
+
+            var getResponse = client.send(getRequest, BodyHandlers.ofString());
+            System.out.println("Read a token:");
+            System.out.println(getResponse.body());
+        } catch (Exception e) {
+            System.out.println(e);
+            System.out.println(e.getMessage());
+            e.printStackTrace();
+        }
+    }
+}
+```
+
+## What can I do next?
+
+Now that you understand the basics, you are ready to learn more about how you can better secure sensitive data without sacrificing data usability using the Basis Theory platform.
+
+Check out the ability to use your [Token data with HTTP request](https://developers.basistheory.com/guides/use-token-data-in-http-requests/) without the data ever touching your systems — or if you’re looking to secure Credit Card data, check out our guide on [How To Charge a customer with Stripe](https://developers.basistheory.com/guides/collect-atomic-cards-with-elements/) while retaining access to the credit card number for future transactions.
+
+Was this useful, or looking for something different? Reach out to us on our [Discord](https://discord.gg/XjWsy8PqK2).
```

Comment:
I think we are not going to use discord anymore?

Reply:
@armsteadj1 what do you think here? are we ready to put our Slack link out there, or wait and update all the docs at the same time early next week?